tests/tests-asn1c-smoke/check-asn1c-smoke.sh

```diff
@@ -8,7 +8,7 @@ top_builddir=${top_builddir:-../..}
 top_srcdir=${top_srcdir:-../..}
 
 cleanup() {
-    rm -rf *.[acho] Makefile.am.* *.txt *.asn
+    rm -rf *.[acho] *.mk *.am *.txt *.asn
```

Comment:
This `rm` will clobber the main `Makefile.am` from which this script is being run. Suggest `lib*am` instead, or something to that effect.

Reply:
I will change to `Makefile.am.* *.mk` since I'll be reverting the Makefile.am.libasncodec name change